force_wfmanager/left_side_pane/data_source_model_view.py

```diff
@@ -122,6 +122,12 @@ class DataSourceModelView(ModelView):
     #: Defines if the evaluator is valid or not
     valid = Bool(True)
 
+    #: An error message for issues in this modelview
+    error_message = Str
```

Comment:
While technically there's no difference in using Str() vs Str, in practice I favour consistency of using () every time, as it's mandatory when you pass information. Also, traitlets (similar to traits) always uses (), so it's good practice that you don't get too attached to that pattern (we do have some code that uses traitlets)